### PR TITLE
#18 daily-standup

### PR DESCRIPTION
**Pull request by HebeChe**
commit for ticket：https://github.com/DevopsChina/openpracticelibrary-cn/issues/18

### CODE REVIEW

**Comment by HebeChe** [comment]
some further materials could not be viewed, I removed them from my translated file, such like IBM Daily Standup with YouTube link https://www.youtube.com/watch?v=GzQjGhD5tSU

**Comment by martinliu** [comment]
> some further materials could not be viewed, I removed them from my translated file, such like IBM Daily Standup with YouTube link https://www.youtube.com/watch?v=GzQjGhD5tSU

I will transfer it to Bilibili and we will add a bilibili shortcode of it.

File: content/posts/daily-standup/daily-standup.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+categories: 
+  - foundation
+resources:
+  - name: featured-image
+    src: dsu.jpg
+  - name: featured-image-preview
+    src: dsu.jpg  
+---
+
+每日站会是由团队统一节奏的，在固定的时间发生的，帮助团队内部快速同步进展的实践活动
```
Comment (martinliu) [inline content/posts/daily-standup/daily-standup.md:19]: ‘由’
这个字会不会多余了一点。

File: content/posts/daily-standup/daily-standup.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+<!--more-->
+
+## 难度
+
+> 中等
+
+## 参与者
+  - Product Owner
```
Comment (martinliu) [inline content/posts/daily-standup/daily-standup.md:27]: Product Owner  -- > 产品负责人 

术语表请先参考：https://github.com/DevOps-Handbook/DevOps-Handbook.github.io/blob/master/glossary.md

File: content/posts/daily-standup/daily-standup.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  * 会议是对整体目标和方向的加强，鼓励团队继续围绕目标向前探索。
+  
+## 如何操作
+
+### 建议
+  * 如上所述，每日站会的主要目的是目标对齐。注意不要让会议演变为状态更新会，聚焦在你需要知道的内容上，把不相干的沟通讨论放到会议之后进行。
+### 远程工作技巧
+  * 打开摄像头，确保每个人都参与进来。
+  * 使用视频会议工具，团队成员分开拨入。--如果有些人员在相同地点办公，让他们一起拨入会议是有些困难的。
+  * 使用线上可视化的计时器，确保站会时长控制在15min之内。核心会议议程：
```
Comment (martinliu) [inline content/posts/daily-standup/daily-standup.md:55]: 15min 翻译成 15 分钟

尽可能翻译成全中文，在必要的地方留下英文（避免歧义）

File: content/posts/daily-standup/daily-standup.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+       1. 昨天我做了什么？
+       2. 今天我计划做什么？
+       3. 我遇到了哪些阻塞？
+  * 人们有可能会在会议上做一些社交类沟通或工作状态的更新，可以在会后留5min用于这类沟通，从而让会议更聚焦。
+  * 如果团队成员分布在不同时区，可以选择一个大家统一的时间组织会议。
+  * 站会后需要讨论的议题，统一记录到阻塞看板或者看板阻塞区。
+  
+  
+## 延伸资源
+- [每日站会提醒音乐播放列表](https://www.youtube.com/playlist?list=PLnqlDDtSH2A7jzZiYPMHy2HjjjdV3rhNM&jct=ykj79fMNekbkUwePDbdsjY2QXKaeag)
```
Comment (martinliu) [inline content/posts/daily-standup/daily-standup.md:65]: 我们会把这里的几个YouTube视频搬运到B站，然后在插入回这里。

File: content/posts/daily-standup/daily-standup.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+- [每日站会提醒音乐播放列表](https://www.youtube.com/playlist?list=PLnqlDDtSH2A7jzZiYPMHy2HjjjdV3rhNM&jct=ykj79fMNekbkUwePDbdsjY2QXKaeag) 
+- [IBM 每日站会](https://www.youtube.com/watch?v=JSFfyse_EXM)
+- [推进远程会议效率的小手段](https://openpracticelibrary.com/blog/facilitation-tips-for-remote-sessions/)
+- [走读看板 - 高效组织日站会的实践模式](https://www.youtube.com/watch?v=TOIccUxlqF0)
+
+
+
+> 原文作者
+- 作者：Matt Takane，Tim Beattie等
+- 发布日期：AUGUST 10, 2018
```
Comment (martinliu) [inline content/posts/daily-standup/daily-standup.md:74]: 参考下面的示例代码

```
> 原文作者

![](https://github.com/prakritiverma.png)

- Prakriti Koller
- PUBLISHED JUNE 29, 2019

> 原文链接：<https://openpracticelibrary.com/practice/lean-coffee/>
```

在原文链接的页面中尽量找到作者头像。

**Review by martinliu — CHANGES_REQUESTED** [review]
增加了一些改进建议。
YouTube的视频会稍后发到评论中。

**Review by martinliu — CHANGES_REQUESTED** [review]
增加了一些改进建议。
YouTube的视频会稍后发到评论中。

File: content/posts/daily-standup/daily-standup.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
+---
```
Comment (martinliu) [inline content/posts/daily-standup/daily-standup.md:1]: 文件名需要修改
 daily-standup.md  --> index.zh-cn.md

File: content/posts/daily-standup/daily-standup.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+templateKey: practice-page
```
[the remark this answers is not in the document]
Comment (HebeChe) [reply content/posts/daily-standup/daily-standup.md:2]: Done, without "link", as I do not have personal blog yet.

**Comment by martinliu** [comment]
视频的短代码如下，加到扩展资源部分。
```
{{< bilibili BV1Jk4y1j7UM >}}

{{< bilibili BV1PP41117ZH >}}
```

File: content/posts/daily-standup/daily-standup.md (hunk 1 above)
Comment (HebeChe) [reply content/posts/daily-standup/daily-standup.md:19]: Done.

File: content/posts/daily-standup/daily-standup.md (hunk 2 above)
Comment (HebeChe) [reply content/posts/daily-standup/daily-standup.md:27]: Done

File: content/posts/daily-standup/daily-standup.md (hunk 3 above)
Comment (HebeChe) [reply content/posts/daily-standup/daily-standup.md:55]: Got it and Done.

File: content/posts/daily-standup/daily-standup.md (hunk 4 above)
Comment (HebeChe) [reply content/posts/daily-standup/daily-standup.md:65]: Got it

File: content/posts/daily-standup/daily-standup.md (hunk 5 above)
Comment (HebeChe) [reply content/posts/daily-standup/daily-standup.md:74]: Done. Only keep 1 author, as there were 5 authers for this practice.  Reader could get them from raw link below.

File: content/posts/daily-standup/daily-standup.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (martinliu) [reply content/posts/daily-standup/daily-standup.md:2]: it's ok.

File: content/posts/daily-standup/daily-standup.md (hunk 6 above)
Comment (HebeChe) [reply content/posts/daily-standup/daily-standup.md:1]: Done

**Review by martinliu — APPROVED** [review]
LGTM

**Comment by 8ar10der** [comment]
After review and approve it.

**Pull request merged**